.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Work Item
+AB#ItemID
```

Comment:
Is it possible to let this template apply to `v3` branch only? `dev` branch is for community usage and it seems this template doesn't fit.

Reply:
@herohua  I have investigated the docs of Github, it only supports the default branch.  
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository 

Or we could make the **Work Item** section optional by 
<pre>
## Work Item ( Optional, please remove this section if the PR is from communitiy )
AB#ItemID

</pre>

Reply:
Then it'll be a bit tricky if we enable this template for `dev` branch...